settings/base.py

```diff
@@ -337,3 +337,5 @@ def is_environ_true(name):
 EMAIL_HOST_PASSWORD = os.environ.get('MAILGUN_SMTP_PASSWORD', '')
 EMAIL_PORT = os.environ.get('MAILGUN_SMTP_PORT', 25)
 EMAIL_USE_TLS = is_environ_true('EMAIL_USE_TLS')
+
+POSTAL_CODE_RESPONSE_CACHE_TIMEOUT = os.environ.get('CACHE_TIMEOUT', 60 * 5)
```

Comment:
I don't think this should be here. If we are going to change the timeout, we're going to do so by changing code. This can live directly in the ``registration/views.py``